chemprop/uncertainty/evaluator.py

```diff
@@ -72,39 +96,55 @@ class ExpectedNormalizedErrorEvaluator(RegressionEvaluator):
     and comparing the average predicted variance of the clusters against the RMSE of the cluster. [1]_
 
     .. math::
-        ENCE = \frac{1}{K} \sum_{i=1}^{K} \frac{|\text{mVAR}(i) - RMSE(i)|}{\text{mVAR}(i)}
+        \text{ENCE} = \frac{1}{N} \sum_{i=1}^{N} \frac{|\mathrm{RMV}_i - RMSE_i|}{\mathrm{mVAR}_i}
```

Comment:
```suggestion
        \mathrm{ENCE} = \frac{1}{N} \sum_{i=1}^{N} \frac{|\mathrm{RMV}_i - RMSE_i|}{\mathrm{RMV}_i}
```

my bad on the prior edits